tests/components/props_validation.test.ts

```diff
@@ -723,6 +723,39 @@ describe("props validation", () => {
     expect(error!).toBeDefined();
     expect(error!.message).toBe("Missing props 'mandatory' (component 'Child')");
   });
+
+  test("can specify that additional props are allowed (array)", async () => {
+    class Child extends Component {
+      static props = ["message", "*"];
+      static template = xml`<div>hey</div>`;
+    }
+    class Parent extends Component {
+      static template = xml`<Child message="'m'" otherProp="'o'"/>`;
+      static components = { Child };
+    }
+
+    await expect(async () => {
+      await mount(Parent, fixture, { dev: true });
+    }).not.toThrowError();
```

Comment:
I don't think it works like this. You need .resolves, but apparently .resolves doesn't work with async functions, only .rejects

I'll dig around to see how to test this without it being a pain, in the meantime, what you can do is:
```js
await expect(mount(Parent, fixture, { dev: true })).resolves.toBe(expect.anything());
```

(or just await mount(...) really, if that throws the test will fail though I understand that it doesn't really document that we're asserting that it doesn't throw)